### DISCUSSION TITLE
Add an unnumbered chapter before the introduction

### DISCUSSION THREAD

**Discussion by ricardo-santos-fhp** [joaomlourenco/novathesis#515] [Q&A] [answered]
Hi,

First of all thank you for the great work with this package!
I'd like to include a prologue/preface in my thesis. I think it cannot be included in the front matter (and don't know how), so I thought it would be an option to add an initial unnumbered chapter.
However, if I do \chapter*{x}, it doesn't show in the table of contents, and the header of the second page is ACRONYMS. Is there a way to overcome this, i.e., have it in the Contents section but without a section number (Introduction remains as Chapter 1)? 

Thanks!

**Comment by joaomlourenco** [comment, marked as answer]
@ricardo-santos-fhp 
Please try:
```latex
\chapter*{Preface}
\addcontentsline{toc}{chapter}{Preface}
\markboth{Preface}{Preface}
```

**Comment by ricardo-santos-fhp** [comment]
It works! Obrigado!

**Reply by joaomlourenco** [reply]
Please give a star to the project.
And, when you finish, a small donation would be welcome! 😊
